From ba1d7cf47b2bbef3a2ccdbb242d8dc4502a4481e Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 26 Jan 2015 13:57:44 +0100
Subject: [PATCH 30/56] src: use cached copy of safebrowsing lists at
 iridiumbrowser.de

To avoid contacing Google, let the UA use safebrowsing lists hosted at
iridiumbrowser. The Iridium project re-gathers these lists on a
regular basis.
---
 chrome/browser/safe_browsing/client_side_model_loader.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrome/browser/safe_browsing/client_side_model_loader.cc b/chrome/browser/safe_browsing/client_side_model_loader.cc
index 863f820..a37dc8c 100644
--- a/chrome/browser/safe_browsing/client_side_model_loader.cc
+++ b/chrome/browser/safe_browsing/client_side_model_loader.cc
@@ -31,7 +31,7 @@ namespace safe_browsing {
 const size_t ModelLoader::kMaxModelSizeBytes = 150 * 1024;
 const int ModelLoader::kClientModelFetchIntervalMs = 3600 * 1000;
 const char ModelLoader::kClientModelUrlPrefix[] =
-    "https://ssl.gstatic.com/safebrowsing/csd/";
+    "https://cache.iridiumbrowser.de/";
 const char ModelLoader::kClientModelNamePattern[] =
     "client_model_v5%s_variation_%d.pb";
 const char ModelLoader::kClientModelFinchExperiment[] =
-- 
2.4.3

